e2e: add baklava config for easier testing

karlb opened the pull request:
Run tests against baklava with `NETWORK=baklava ./run_all_tests.sh` after giving the wallet enough balance (or setting your own).

karlb committed:
e2e: add baklava config for easier testing

Run tests against baklava with `NETWORK=baklava ./run_all_tests.sh`
after giving the wallet enough balance (or setting your own).

palango reviewed (approved):
Nice

piersy reviewed (approved):
Looks good to me, tests passed on our baklava dry run when I port forwarded to a single node.

karlb committed:
e2e_test: set tip to at least 1

The tip must be at least native 1 wei for the tx to be included. No
matter what the exchange rate for a fee currency is, if the fee currency
tip is zero, we can't reach 1 wei if the fee currency tip is zero. So
increase the tip to at least one.

karlb commented:
I know that @piersy prefers squashes so that we have the PR number in the commit, but I still think doing a merge-by-rebase would be nicer for PRs like this where the commits would be squashed with different commits in the next history rewrite before upstream-rebase. So I would vote for having both options enabled in this repo.

palango commented:
> I know that @piersy prefers squashes so that we have the PR number in the commit, but I still think doing a merge-by-rebase would be nicer for PRs like this where the commits would be squashed with different commits in the next history rewrite before upstream-rebase. So I would vote for having both options enabled in this repo.

I enabled rebase merging now

piersy commented:
> at we have the PR number in the commit, but I still think doing a merge-by-rebase would be nicer for PRs like this where the commits would be squashed with different commits in the next history rewrite before upstream-rebase. So I would vote for having both options enabled in this repo.

That's a good reason
